gravitee-am-ui/package.json

```diff
@@ -77,23 +77,33 @@
     "zone.js": "0.11.8"
   },
   "devDependencies": {
-    "@angular-builders/jest": "11.2.0",
-    "@angular-devkit/build-angular": "13.3.11",
-    "@angular/cli": "13.3.11",
-    "@angular/compiler-cli": "13.4.0",
-    "@angular/language-service": "13.4.0",
-    "@types/jest": "25.2.3",
+    "@angular-builders/jest": "14.1.0",
+    "@angular-devkit/build-angular": "14.2.13",
+    "@angular-eslint/builder": "14.4.0",
+    "@angular-eslint/eslint-plugin": "14.4.0",
+    "@angular-eslint/eslint-plugin-template": "14.4.0",
+    "@angular-eslint/schematics": "14.4.0",
+    "@angular-eslint/template-parser": "14.4.0",
+    "@angular/cli": "14.2.13",
+    "@angular/compiler-cli": "14.3.0",
+    "@angular/language-service": "14.3.0",
+    "@storybook/addon-essentials": "6.4.17",
+    "@storybook/angular": "6.4.17",
+    "@types/jest": "28.1.8",
     "@types/lodash": "4.14.196",
     "@types/node": "12.20.55",
+    "@typescript-eslint/eslint-plugin": "5.43.0",
+    "@typescript-eslint/parser": "5.43.0",
     "@webcomponents/webcomponentsjs": "2.8.0",
     "codelyzer": "6.0.2",
-    "jest": "26.6.3",
+    "eslint": "^8.28.0",
+    "eslint-import-resolver-typescript": "^3.6.1",
+    "jest": "28.1.3",
+    "jest-preset-angular": "^12.2.6",
     "node-polyfill-webpack-plugin": "1.1.4",
     "protractor": "7.0.0",
     "ts-node": "8.2.0",
```

Comment:
Should ts-node be updated?

[v10.4.0](https://github.com/TypeStrong/ts-node/releases/tag/v10.4.0)

"Adds support for targets "es2020" and "es2021"...."

Reply:
Yes. it seems to me that he's only here for jest.  As jest is not used here, I can do it later. With a dedicated subject to make test working agin on this project